.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Bug report
+about: Help us make Pants better
```

Comment:
This sounds very similar to feature request. Maybe "more reliable"? Or "Help us squash bugs"?